Formula/dagger.rb

```diff
@@ -1,7 +1,7 @@
 class Dagger < Formula
   desc "Portable devkit for CI/CD pipelines"
   homepage "https://dagger.io"
-  url "https://github.com/dagger/dagger.git",
+  url "https://github.com/dagger/dagger/archive/v0.2.28.tar.gz",
```

Comment:
This is a follow-up to https://github.com/Homebrew/homebrew-core/pull/105746#issuecomment-1190551398. We are in the process of automating these PRs from our pipeline. I can see that this change looks wrong. We should have updated the tag & revision. Let us try one more time and see if we can do this better.

In the meantime, it's OK to close this.

Thanks!

cc @SMillerDev

Reply:
If you run `brew bump-formula-pr` or use https://github.com/dawidd6/action-homebrew-bump-formula you won't have to reinvent the wheel here